gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/SpecStoreChangeMonitor.java

```diff
@@ -186,7 +185,8 @@ protected void processMessage(DecodeableKafkaRecord message) {
         return;
       }
     } catch (Exception e) {
-      log.warn("Ran into unexpected error processing SpecStore changes. Reexamine scheduler. Error: {}", e);
+      log.warn("Ran into unexpected error processing specUri {} changes. Reexamine scheduler. "
```

Comment:
adding some logs to debug some exceptions coming from `this.flowCatalog.getSpecWrapper()`

Reply:
makes sense...

nit: no need for "Ran into "